src/Http/Controllers/AmastyShopByBrandController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rapidez\AmastyShopByBrand\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Rapidez\Core\Facades\Rapidez;
+use Rapidez\AmastyShopByBrand\Resolvers\BrandResolver;
+
+class AmastyShopByBrandController
+{
+    public function __invoke(Request $request)
+    {
+        $resolver = app(BrandResolver::class);
```

Comment:
You could also dependency inject this in your function arguments, if you're not using $request you don't need to include it in the arguments